src/foam/nanos/extraconfig/Config.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.extraconfig',
+  name: 'Config',
+  properties: [
+    {
+      name: 'configValueString',
+      class: 'String',
+      value: ''
+    },
+    {
+      name: 'exportMetadata',
+      class: 'FObjectProperty',
+      of: 'foam.nanos.extraconfig.AddOn'
+    },
+    {
+      name: 'configValue',
+      factory: function() {     
+          if ( this.exportMetadata.typeOfConfig === 'Boolean' )
+            return false;
+          else if ( this.exportMetadata.typeOfConfig === 'Integer' || this.exportMetadata.typeOfConfig === 'Float' )
```

Comment:
No need for the 'else if' if you've already returned above. Just an 'if' is sufficient.

Reply:
fixed